daemon/src/engine/server/sv_ccmds.cpp

```diff
@@ -369,6 +385,48 @@ static void SV_Systeminfo_f()
 	Info_Print( Cvar_InfoString( CVAR_SYSTEMINFO, false ) );
 }
 
+class ListMapsCmd: public Cmd::StaticCmd
```

Comment:
iirc, this is implemented in the gamelogic, and this command will overwrite that. It should be available over rcon as well. If you decide that this is better, please delete the gamelogic function from sg_cmds.cpp:Cmd_ListMaps_f()